app/src/main/java/com/pajato/android/gamechat/event/GroupJoinedEvent.java

```diff
@@ -3,21 +3,13 @@
 import java.util.List;
 
 /**
- * Provides a event class indicating that the current user has been joined to one or more groups.
+ * Provides an event class indicating that the current user has been joined to one or more groups.
  */
 public class GroupJoinedEvent {
 
-    /**
-     * The name of the group which has been joined. If the group is null, more than one group
-     * has been joined.
-     */
+    /** Null if more then one group joined; otherwise the name of the joined group */
     public String groupName;
 
-    /**
-     * Build the event and indicate the specified group. If the group is null, more than one
-     * group has been joined.
-     */
-    public GroupJoinedEvent(String grp) {
-        groupName = grp;
-    }
+    /** Build the event and indicate the specified group (or null) */
+    public GroupJoinedEvent(String grp) { groupName = grp; }
```

Comment:
Couple of things: 1) to facilitate setting breakpoints, I now use less one-liners; 2) grp can be groupKey to be a tad more clear without requiring an additional line.